RUFAS/routines/EEE/emissions.py

```diff
@@ -309,6 +338,12 @@ def _calculate_homegrown_feed_emissions(
             aggregated_manure_apps[field_name]["nitrogen"] += app["nitrogen"]
             aggregated_manure_apps[field_name]["phosphorus"] += app["phosphorus"]
 
```

Comment:
```suggestion
fields_with_manure_requests = {request["field_name"] for request in manure_requests}
all_fields = list(fields_with_manure_requests | fields_with_crops)
```
This handles the edge case where there was a field that grew crops but did not request any manure.